docs/conf.py

```diff
@@ -0,0 +1,13 @@
+# outline for a myst_nb project with sphinx
+# build with: sphinx-build -nW --keep-going -b html . ./_build/html
```

Comment:
This should be defined as a pixi task
https://pixi.sh/latest/features/advanced_tasks/